ui/ducks/send/send.js

```diff
@@ -2378,12 +2383,20 @@ export function signTransaction() {
         ),
       );
 
-      dispatch(
+      const result = await dispatch(
         addTransactionAndRouteToConfirmationPage(txParams, {
           sendFlowHistory: draftTransaction.history,
           type: transactionType,
         }),
       );
+      dispatch({
+        type: SET_DETAILS_FOR_CONFIRM_TX,
+        payload: {
+          amountMode,
```

Comment:
As mentioned in the duck, since `amountMode` and `assetType` are specific to this duck, could we instead check them now and simply send a `boolean` in the action to decouple the two ducks?

Reply:
Done in https://github.com/MetaMask/metamask-extension/commit/f362be605aa5deda59e74b27bfe810a05b60db8e